src/api.v2/controllers/experimentController.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/prefer-default-export */
+const config = require('../../config');
+
+const experiment = require('../model/experiment');
+const userAccess = require('../model/userAccess');
+
+const AccessRole = require('../../utils/enums/AccessRole');
+
+const { OK } = require('../../utils/responses');
+
+const createExperiment = async (req, res) => {
```

Comment:
can you import the logger in here and log some messages that will be useful for us for debugging? Like "Creating experiment ${experimentId}" and "${experimentId} created successfully" and so on?